tools/cmake/mbed_greentea.cmake

```diff
@@ -30,19 +35,23 @@ macro(mbed_greentea_add_test)
         TEST_INCLUDE_DIRS
         TEST_SOURCES
         TEST_REQUIRED_LIBS
+        HOST_TESTS_DIR
```

Comment:
Does this argument need to be added to the mbed_greentea_add_test documentation comment?

Reply:
Good spot. Yes, I'll add it to the comment.